source/guides/recipes/dependency-management.md

```diff
@@ -7,7 +7,7 @@ Nix expressions themselves can depend on remote sources, and there are multiple
 For more automation around handling remote sources, set up [niv](https://github.com/nmattia/niv/) in your project:
 
 ```shell-session
-$ nix-shell -p niv --run "niv init --nixpkgs nixos/nixpkgs --nixpkgs-branch nixos-23.05"
+$ nix-shell -p niv --run "niv init --nixpkgs nixos/nixpkgs --nixpkgs-branch nixos-23.11"
 ```
 
 This command will fetch the latest revision of the Nixpkgs 23.05 release branch.
```

Comment:
```suggestion
This command will fetch the latest revision of the Nixpkgs 23.11 release branch.
```

Reply:
That section was talking about using an older version of nixpkgs so 23.05 would be correct since 23.11 is the current release.

Reply:
```
As an example, we will use the previously created expression with an older version of Nixpkgs.

Enter the development environment, create a new directory, and set up niv with a different version of Nixpkgs:
```

Reply:
I don't think that is correct. Please note, the suggested edit is on row 13, which refers to the command defined on row 10. 

Your snapshot above are row 61 and 63 and belong to a different section.

Screenshots below hopefully help to clarify it.

![image](https://github.com/NixOS/nix.dev/assets/569510/d840cfbf-1f3c-4540-9cce-c5e15ab4aa59)

![image](https://github.com/NixOS/nix.dev/assets/569510/bd68fbe3-a3b8-4c1a-850e-9c85bb401d68)